tests/acceptance/SettingsPageBasicsCest.php

```diff
@@ -53,6 +53,7 @@ function allowSubscribeInComments(\AcceptanceTester $I) {
     $I->amOnPage('/');
     $I->waitForText($post_title);
     $I->click($post_title);
+    $I->scrollTo('.comment-form-mailpoet');
```

Comment:
Why is this necessary? Also - is the element available for scroll even before `waitForElement`?

Reply:
That's b/c of a quirk I've noticed in Codeception. I checked the artifacts on that failing test, and the mailpoet comment form was showing in the HTML, but not in the screenshot (it was out of frame) - adding a scrollTo makes it consistently pass for me locally and on circle. It fails about 1/4 without it.